jobs/service-fabrik-broker/monit

```diff
@@ -1,6 +1,6 @@
 
 check process service-fabrik-broker with pidfile /var/vcap/sys/run/service-fabrik-broker/service-fabrik-broker.pid
-  group vcap<% job_name = p('docker.job', nil); if p('enable_swarm_manager', true) && (!job_name.to_s.empty?) then %>
+  group vcap<% job_name = p('docker.job', nil); if p('ha_enabled', false) != true && p('enable_swarm_manager', true) && (!job_name.to_s.empty?) then %>
   depends on <%= job_name %><% end %>
```

Comment:
how does it matter if swarm is in the same box or diff box. Why is this dependency not required when running on different hosts?

Reply:
When swarm runs on different host then 'swarm_manager' job itself doesn't reside under the broker vm. then this dependency not required.